src/widget/systemtrayicon.cpp

```diff
@@ -26,8 +26,9 @@
 #include <QDebug>
 #include "src/persistence/settings.h"
 
-SystemTrayIcon::SystemTrayIcon()
+SystemTrayIcon::SystemTrayIcon(Widget *parent)
```

Comment:
that was used to experiment with qt implementation/status notifier, in fact is should be removed because parent is never used anyway. Thanks for spotting this o/